lib/3scale_toolbox/commands/update_command/update_service.rb

```diff
@@ -34,9 +35,9 @@ def self.fetch_required_option(options, key)
         end
 
         class ServiceUpdater
-          attr_reader :source_client, :target_client, :source_service_id, :target_service_id
+          attr_reader :source_client, :target_client, :source_service_id, :target_service_id, :target_system_name
 
-          def initialize (source, source_service_id, destination, target_service_id, insecure)
+          def initialize (source, source_service_id, destination, target_service_id, insecure, system_name)
```

Comment:
This starts to be a bit too much.

Reply:
Sure! 
refactor is coming with new features of the CLI framework :)